stories/src/primatives/author-list.jsx

```diff
@@ -20,8 +20,6 @@ const Wrapper = styled.div`
   user-select: none;
   transition: background-color 0.1s ease;
   margin: ${grid}px 0;
-
-  overflow: auto;
```

Comment:
We had two levels of scroll containers in our horizontal list stories. Removing this to reduce confusion.